sim/common/cata/other_effects.go

```diff
@@ -825,6 +836,253 @@ func init() {
 			})
 		})
 	}
+
+	for version, _ := range []ItemVersion{ItemVersionLFR, ItemVersionNormal, ItemVersionHeroic} {
```

Comment:
If you make the `ItemVersion` change, then this would be instead be
```
for _, version, := range []ItemVersion{ItemVersionLFR, ItemVersionNormal, ItemVersionHeroic} {
```